extensions/wikia/ARecoveryEngine/js/InstartLogic/recovery.js

```diff
@@ -0,0 +1,38 @@
+define('ext.wikia.aRecoveryEngine.instartLogic.recovery', [
+	'ext.wikia.adEngine.adContext',
+	'wikia.log',
+	'wikia.window'
+], function (adContext, log, win) {
+	'use strict';
+
+	var logGroup = 'ext.wikia.aRecoveryEngine.instartLogic.recovery',
+		context = adContext.getContext();
+
+	/**
+	 * When IL detects that user is blocking, it sets src=rec on page level params
+	 *
+	 * @returns {boolean}
+	 */
+	function isBlocking() {
+		var srcTargeting = win.googletag.pubads().getTargeting('src'),
+			isBlocking = srcTargeting && srcTargeting.indexOf('rec') > -1;
+
+		debugger
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Remove this debugger statement. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=javascript%3ADebuggerStatement)